client/charts/js/components/HomepageSelection.jsx

```diff
@@ -111,14 +124,10 @@ export default function HomepageSelection({
 			/>
 			<ButtonsRow
 				label="from"
-				buttonLabels={['the past six months'].concat(years)}
+				buttonLabels={['the past six months'].concat(years).concat(['All time'])}
```

Comment:
I feel the below makes it cleaner/more readable?
```suggestion
				buttonLabels={['the past six months', 'All time'].concat(years)}
```

Reply:
I like this